content/languages/lambdacalc/authoring.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Authoring Lambda Calculus Content
+sidebar_label: Authoring
+---
+
+This article is intended for kata authors and translators who would like to create new content in Lambda Calculus. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls and how to avoid them.
+
+This article is not a standalone tutorial on creating kata or translations. It's meant to be a complementary, Lambda Calculus-specific part of a more general set of HOWTOs and guidelines related to [content authoring](/authoring/). If you are going to create a Lambda Calculus translation, or a new Lambda Calculus kata from scratch, please make yourself familiar with the aforementioned documents related to authoring in general first.
+
+
+## General info
+
+Any technical information related to the Lambda Calculus setup on Codewars can be found on the [Lambda Calculus reference](/languages/factor/) page (language versions, available libraries, and setup of the code runner).
+
+For further information about Lambda Calculus as a language, and for a collection of extensive authoring, syntax and style guides, see [The Lambda Calculus Wiki](https://github.com/JohanWiltink/lc-docs/wiki).
+
+## Description
+
+Lambda Calculus code blocks can be inserted with Lambda Calculus-specific part in [sequential code blocks](/references/markdown/extensions/#sequential-code-blocks):
+
+~~~
+```lambdacalc
+
+...your code here...
+
+```
+~~~
+
+Lambda Calculus-specific paragraphs can be inserted with [language conditional rendering](/references/markdown/extensions/#conditional-rendering):
+
+```
+~~~if:lambdacalc
+
+...text visible only for Lambda Calculus description...
+
+~~~
+
+~~~if-not:lambdacalc
+
+...text not visible in Lambda Calculus description...
+
+~~~
+```
+
+## Tasks and Requirements
+
+Due to being an esoteric language, in many cases tasks may be significantly harder to complete in Lambda Calculus, than in other languages. Due to this, many kata translations should rather be published as standalone katas with `[LC]` as a title prefix.
+
+## Tests
+
+### lc-test
+
+Lambda Calculus kata use [lc-test](/languages/lambdacalc/lc-test/) to implement and execute tests.
+
+### Setup
+
+Lambda Calculus is heavily dependant upon what encodings are chosen to represent various data types. All Lambda Calculus kata should declare in the description which `Purity` and which `numEncoding` (if any) are going to be used in the kata. Additionally, any required encodings should also be described, such as expected input and output types.
```

Comment:
Mention that encodings are specified by constructors and deconstructors, precisely to leave the actual implementation free.